test/unit/senderSpec.js

```diff
@@ -73,25 +81,6 @@ describe('UNIT Sender', function () {
       }, 10);
     });
 
-    it('should not override internal request params if passed into constructor (except timeout/uri)', function (done) {
```

Comment:
Unfortunately we will need this test to pass, we won't be able to merge with the test deleted.